app/views/userProfile.scala.html

```diff
@@ -14,22 +14,26 @@
             <div class="row" id="profile-container">
                 <div id="username">@user.get.username</div>
                 <div id="org-container">
-                    @if(!UserOrgTable.getAllOrgs(user.get.userId).isEmpty) {
-                        <div id="org-name">@OrganizationTable.getOrganizationName(UserOrgTable.getAllOrgs(user.get.userId).head)</div>
-                    } else {
-                        <div id="org-name">@Messages("dashboard.no.org")</div>
-                    }
-                    <div id="edit-org-dropdown" class="dropdown" data-toggle="popover" data-trigger="hover" data-placement="right" data-content="@Messages("dashboard.edit.info")">
-                        <a id="edit-org-button" role="button" data-toggle="dropdown" href="#">@Messages("dashboard.edit.org") <b class="caret"></b></a>
+                    <div id="edit-org-dropdown" class="dropdown" data-toggle="popover" data-trigger="hover" data-placement="right">
+                        @if(!UserOrgTable.getAllOrgs(user.get.userId).isEmpty) {
+                            <a id="edit-org-button" role="button" data-toggle="dropdown" href="#">
+                                @OrganizationTable.getOrganizationName(UserOrgTable.getAllOrgs(user.get.userId).head) <b class="caret"></b>
+                            </a>
+                        } else {
+                            <a id="edit-org-button" role="button" data-toggle="dropdown" href="#">
+                                @Messages("dashboard.join.team") <b class="caret"></b>
+                            </a>
+                        }
                         <ul id="edit-org-menu" class="dropdown-menu" role="menu">
-                            @if(OrganizationTable.getAllOrganizations.isEmpty) {
-                                <li class="menu-info">@Messages("dashboard.orgs.unavailable")</li>
-                            }
                             @if(!UserOrgTable.getAllOrgs(user.get.userId).isEmpty) {
                                 <li>
-                                    <a id="from-@UserOrgTable.getAllOrgs(user.get.userId).headOption.getOrElse(0)-to-0" role="menuitem" class="put-user-org">@Messages("dashboard.leave.org")</a>
+                                    <a id="from-@UserOrgTable.getAllOrgs(user.get.userId).headOption.getOrElse(0)-to-0" role="menuitem" class="put-user-org"><Strong>@Messages("dashboard.leave.team") @OrganizationTable.getOrganizationName(UserOrgTable.getAllOrgs(user.get.userId).headOption.getOrElse(0))</Strong></a>
```

Comment:
And we should probably be structuring it like
```
<a>
    <strong></string>
</a>
```

Instead of
```
<a><strong></strong></a>
```

Sometimes the lines are going to be longer than 120 characters, but this 317 character line is over the top :joy: